tools/automation.py

```diff
@@ -143,6 +144,40 @@ def get_mentorship_type(mentorship_type_str):
 
     return result
 
+def add_availability(months_str):
+    """
+    Convert a comma-separated string of month names to a list of month numbers.
+    If the string is empty, return an empty list.
+    """
+
+    month_map = {
+        'january': 1,
+        'february': 2,
+        'march': 3,
+        'april': 4,
+        'may': 5,
+        'june': 6,
+        'july': 7,
+        'august': 8,
+        'september': 9,
+        'october': 10,
+        'november': 11,
+        'december': 12
```

Comment:
December is not an option, also we start in April. That should be the available months.